src/styles/_app.scss

```diff
@@ -0,0 +1,23 @@
+body {
+  background:linear-gradient(rgba(0,0,0,0.7),rgba(0,0,0,0.7)),url("https://i.imgur.com/0ihenJu.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.home-title {
+  @include title-style();
+  color: white;
+}
+
+.filters {
```

Comment:
The shows buttons look really good gridded. What do you think if we consider centering the select-menu below the show buttons?

Reply:
Yes, I like that idea. I haven't worked on styling that yet because it's in a separate component - it doesn't seem the most efficient using separate .scss files for each component because many of them have shared styles as well. Definitely something worth discussing when we all get together.